.gitignore

```diff
@@ -29,3 +29,4 @@ test/fixtures/rake/rake_tasks/public/assets*
 test/fixtures/rake/rake_tasks_app/public/assets*
 test/fixtures/rake/rake_tasks/db/*.sqlite
 test/fixtures/rake/rake_tasks_app/db/*.sqlite
+*.idea
```

Comment:
I was with vim, but started using rubymine to run some plugins for static code analyzes. :3